scss/_variables.scss

```diff
@@ -837,7 +837,14 @@ $zindex-modal:                      1050 !default;
 $zindex-popover:                    1060 !default;
 $zindex-tooltip:                    1070 !default;
 // scss-docs-end zindex-stack
-
+$z-index-levels: (
+  0,
+  1,
+  2,
+  3,
+  4,
+  5
+) !default;
 
```

Comment:
This would require documenting those new utilities and the Sass map, too.

Reply:
I'm going to work on that

Reply:
I think that z-index-negative is better than z-index--1

Reply:
Good catch; I'd recommend n1 like we do in margins: `z-index-n1`. Does this sound good to you?

Reply:
wooow n1 is better than negative. I agree @ffoodd

Reply:
where can I put the doc of z-index-levels?

Reply:
In the utilities section, as a new page I guess.

Reply:
Okasss

Reply:
I created a stack page for z-index util, sounds good for you?

Reply:
Jesus Christ KING OF KINGS LORD OF LORDS!!!...

Reply:
I love the bootstrap project structure.